shemaeric1234/FreeMentors
# #168234052 add sign up api

#### What does this PR do?
enable a user to sign up

#### Description of Task to be completed?
```
Given a new user

When i visit this url API/V1/auth/signup with POST method
Then I will be registered as a mentee
and logged-in

Dev Notes: users logged in automatically
201 Ok
403 Bad request
```
#### How should this be manually tested?
How should this be manually tested
After clone repository goes to the root of project open path into command run the npm install and once setup is done click npm test.

Using Postman to test all endpoints:
Key: Content-Type value: application/JSON
To test authentication, add this to the header: Bearer TOKEN
Test endpoints
POST request
visit: URL/API/v1/auth/signup

#### What are the relevant pivotal tracker stories?
[#168234052](https://www.pivotaltracker.com/story/show/168234052)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/47531860/64074358-57bcac00-ccaa-11e9-86d4-704c2a946951.png)
